Description: Increases the number of bookmarks visible in Places before overflowing to a submenu
Bug: https://bugs.launchpad.net/ubuntu/+source/gnome-panel/+bug/204567

Index: gnome-panel-3.4.1/gnome-panel/panel-menu-items.c
===================================================================
--- gnome-panel-3.4.1.orig/gnome-panel/panel-menu-items.c	2012-04-19 17:48:06.900533540 +0200
+++ gnome-panel-3.4.1/gnome-panel/panel-menu-items.c	2012-04-19 17:49:48.820532544 +0200
@@ -68,7 +68,7 @@
 #define NAMES_DIR               "/apps/nautilus/desktop"
 #define HOME_NAME_KEY           "/apps/nautilus/desktop/home_icon_name"
 #define COMPUTER_NAME_KEY       "/apps/nautilus/desktop/computer_icon_name"
-#define MAX_ITEMS_OR_SUBMENU    5
+#define MAX_ITEMS_OR_SUBMENU    8
 #define MAX_BOOKMARK_ITEMS      100
 
 G_DEFINE_TYPE (PanelPlaceMenuItem, panel_place_menu_item, GTK_TYPE_IMAGE_MENU_ITEM)
